package-lock.json

```diff
@@ -1715,6 +1722,11 @@
       "integrity": "sha512-Fo79ojj3vdEZOHg3wR9ksAMRz4P3S5fDB5e/YWZiFnyFQI1WY2Vftu9XoXVVtJfxB7Bpce/QTqWSSntkz2Znrw==",
       "dev": true
     },
+    "node_modules/@types/qs": {
```

Comment:
[learning] why do all these node modules get updated?

Reply:
I don't think that it make sense to commit this file anymore, i'll check with our frontend expert on this one (because most likely the file will change if anyone else or me has a fresh install of first time trying to build the solution)

Reply:
Ok, I'll let you decide, feel free to merge the PR if you want to keep it for now